config.make

```diff
@@ -20,7 +20,10 @@ endif
 
 H5Z_ZFP_PLUGIN := $(H5Z_ZFP_BASE)/plugin
 H5Z_ZFP_VERSINFO := $(shell grep '^\#define H5Z_FILTER_ZFP_VERSION_[MP]' $(H5Z_ZFP_BASE)/H5Zzfp_plugin.h | cut -d' ' -f3 | tr '\n' '.' | cut -d'.' -f-3 2>/dev/null)
+ZFP_HAS_REVERSIBLE :=
```

Comment:
Perhaps not that clean, but `ZFP_VERSION >= 0x55` ought to do the trick, no?

Reply:
Probably and will re-think for 1.0.0. Merging this PR for now to accomodate @jwsblokland